play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Chassis.txt

```diff
@@ -0,0 +1,19 @@
+		Light Chassis
+Chassis Name:	Mantis
+Chassis Power Req:	3 PU
+Standard Speed:		1m/action
+Maneuver Bonus:		2m/action
+
+Fort Bonus:		2
+Reflex Penalty:		30
+Perception Penalty:	25
+
+Disrupt DC:		75
+Pwn DC:			120
+
```

Comment:
I suggest we use caution when giving fort bonuses, since that will grant additional death saving throws.

Why are you giving a perception penalty?

Also, where is the number of slots in the chassis stated?